src/Workspaces/CSharp/Portable/LanguageServices/CSharpSyntaxFactsService.cs

```diff
@@ -688,6 +694,8 @@ public bool IsSimpleArgument(SyntaxNode node)
                    argument.NameColon == null;
         }
 
+        public bool IsTypeArgument(SyntaxNode node)
+            => node.IsParentKind(SyntaxKind.TypeArgumentList);
```

Comment:
this would be better as `syntaxFacts.IsTypeArgumentList(node.Parent)`.  a TypeArgumentList is something both languages have, and it's a direct answer about that node.

If you do want to do this check, this should be called something like IsElementOfTypeArgumentList.